### ISSUE TITLE
BCU window opens but doesnt fully load

### ISSUE THREAD

**Issue by bobbbobobb** [Klocman/Bulk-Crap-Uninstaller#475]
so up until recently BCU was working fine, now it just opens the program window but never resolves into a window that comes to the front. i installed the latest update but still same

**Comment by bobbbobobb** [comment]
i downloaded the portable version and that works so i ll just use that. thanks for a very useful product

**Comment by Klocman** [comment]
Try removing the BCUninstaller.settings file from the application directory.

**Comment by Rythoka** [comment]
I experienced the same issue and deleting the settings file fixed it 👍

**Comment by Klocman** [comment]
If anyone has this issue, please send me your config file before removing it.

**Comment by bobbbobobb** [comment]
> If anyone has this issue, please sen
[BCU-console.exe.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/11522461/BCU-console.exe.zip)
d me your config file before removing it.

**Comment by bobbbobobb** [comment]
worked for me too! thanks :-)

**Comment by Klocman** [comment]
@bobbbobobb Sorry, I meant the `BCUninstaller.settings` file. The .config files don't actually contain the user config.

**Comment by bobbbobobb** [comment]
> @bobbbobobb Sorry, I meant the `BCUninstaller.settings` file. The .config files don't actually contain the user config.
ok gotcha, sorry my bad, didnt read your post carefully enough. But the file was updated yesterday when i fixed the problem by deleting the file so may be not much use
[BCUninstaller.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/11525152/BCUninstaller.zip)

**Comment by Klocman** [comment]
Yeah, I need the settings file that causes the bug to happen. Hopefully this will be fixed in v5.6.
